README.md

```diff
@@ -1 +1,275 @@
-# landing-zone-config-tool
\ No newline at end of file
+# Landing Zone Config Tool
+
+This is a companion GUI tool for [IBM Secure Landing Zone](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone), primarily for deployments creating through the [IBM Cloud Catalog](https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vpc-9fc0fa64-27af-4fed-9dce-47b3640ba739-global).
+
+It can help create an override.json file used for configuring your deployment of Secure Landing Zone. It can also be used to modify or validate existing override.json files.
+
+
+Live Version: https://ibm.biz/override-landing-zone
+
+Direct Link: https://setup.deploy-arch-ibm-slz-vpc.cloud.ibm.com
+
+---
```

Comment:
Lets make it clear that the below headings are all for local developer set up and not for consumers